.github/workflows/CI.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        dotnet-version: ["3.1.x", "5.0.x", "6.0.x", "7.0.x"]

```

Comment:
Do we have any other docs that need to be updated to indicate we are no longer supporting these old .NET versions?